.gitignore

```diff
@@ -41,6 +41,7 @@ build/Release
 # Dependency directories
 node_modules/
 jspm_packages/
+package-lock.json/
```

Comment:
Don't ignore it friend. We need this file. Review: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
Done.